store/tikv/scan.go

```diff
@@ -200,6 +200,7 @@ func (s *Scanner) getData(bo *Backoffer) error {
 		req := tikvrpc.NewRequest(tikvrpc.CmdScan, sreq, pb.Context{
 			Priority:     s.snapshot.priority,
 			NotFillCache: s.snapshot.notFillCache,
+			FollowerRead: s.snapshot.replicaRead.IsFollowerRead(),
```

Comment:
`FollowerRead` has been renamed to `ReplicaRead` in the latest kvproto.

Reply:
Sure, let me update dependency.